docs/understanding-tracking-design/json-schema-type-casting-rules/index.md

```diff
@@ -81,11 +108,17 @@ For the purposes of this  `number` and `integer` are the same.
 
   ```json
 {
-    "type": [T1, T2, ...]
+    "type": ["boolean", "integer"]
 }
 ```
 
-- `T1, T2, ..`. only contains `"boolean"` **and** `"integer"`
+OR 
+
+  ```json
+{
+    "type": [ "integer", "boolean"]
+}
```

Comment:
Should `{"type": ["number", "integer"]}` be moved up, e.g. after this line? Otherwise I agree with the feedback that it would be never reached because of `{"type": [T1, T2, ...]}`.

Reply:
it is above that.